packages/xrpl/src/Wallet/index.ts

```diff
@@ -128,6 +128,11 @@ class Wallet {
    * @returns A new Wallet derived from a generated seed.
    */
   public static generate(algorithm: ECDSA = DEFAULT_ALGORITHM): Wallet {
+    if (algorithm !== DEFAULT_ALGORITHM || algorithm !== "ed25519" || algorithm !== "secp256k1") {
```

Comment:
```suggestion
    if (algorithm !== DEFAULT_ALGORITHM || algorithm !== "ed25519" || algorithm !== "ecdsa-secp256k1") {
```
"ecdsa-secp256k1" is the correct value according to https://github.com/XRPLF/xrpl.js/blob/main/packages/xrpl/src/ECDSA.ts

Reply:
You might be able shorten it to `!Object.values(ECDSA).includes(algorithm as ECDSA)`

Reply:
Thanks @ckniffen